cli/types/cypress.d.ts

```diff
@@ -3051,11 +3051,11 @@ declare namespace Cypress {
 
   type DevServerConfigOptions = {
     bundler: 'webpack'
-    framework: 'react' | 'vue' | 'vue-cli' | 'nuxt' | 'create-react-app' | 'next' | 'angular'
+    framework: 'react' | 'vue' | 'vue-cli' | 'nuxt' | 'create-react-app' | 'next' | 'angular' | 'svelte'
```

Comment:
We could shift this to have `cypress.d.ts` be the source of truth and then use something like:
```ts
type WebpackFrameworks = Extract<Cypress.DevServerConfigOptions, { bundler: 'webpack' }>['framework']
```

Otherwise, we'd have to copy the types into CLI postbuild (at least, that's my understanding of it)